lib/src/jpegencoderhelper.cpp

```diff
@@ -198,6 +200,13 @@ uhdr_error_info_t JpegEncoderHelper::encode(const uint8_t* planes[3], const size
     if (iccBuffer != nullptr && iccSize > 0) {
       jpeg_write_marker(&cinfo, JPEG_APP0 + 2, static_cast<const JOCTET*>(iccBuffer), iccSize);
     }
+    if (isGainMapImg) {
+      char comment[255];
+      snprintf(comment, sizeof comment,
+               "Source: google libuhdr v1.1.1, Coder: libjpeg v%d, Attrib: GainMap Image",
```

Comment:
Also use a variable for the version of this lib

Reply:
+1. Thank you for the comment @cmahnke

Reply:
Done